ThunderCloud/UIView+ShadowComponents.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ShadowComponents.swift
+//  ThunderCloud
+//
+//  Created by Simon Mitchell on 01/07/2020.
+//  Copyright © 2020 threesidedcube. All rights reserved.
+//
+
+import Foundation
+
+public extension UIView {
+    
+    /// Component representation of all properties required to render a shadow in UIKit
+    struct ShadowComponents {
```

Comment:
Also nice :D! Sometimes a `shadowPath` should be updated in `layoutSubviews` but that does not stop this being clean and useful